{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -39,5 +39,154 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
         How noisy the logger should be.
 
     """
-    # Stubbed out pending further consideration of best location for this code.
-    pass
+    output_dir = Path(output_dir)
+
+    if location in project_globals.LOCATIONS:
+        path = Path(output_dir) / f'{sanitize_location(location)}.hdf'
+
+        if path.exists() and not append:
```

Comment:
I think we probably want to pull a function to do this for us out.  It seems generally useful.  Something like 
```
def delete_if_exists(*paths: Union[Path, List[Path]], confirm=False):
    paths = paths[0] if isinstance(paths[0], list) else paths    
    existing_paths = [p for p in paths if p.exists()]
    if existing_paths:
        if confirm:
            # Assumes all paths have the same root dir
            root = existing_paths[0].parent
            names = [p.name for p in existing paths]
            click.confirm(f"Existing files {names} found in directory {root}.  Do you want to delete and replace?", abort=True)
        for p in existing paths:
            p.unlink()
```

Reply:
Done.